asf_heat_pump_suitability/pipeline/run_scripts/run_calculate_garden_size.py

```diff
@@ -150,7 +169,13 @@ def parse_arguments() -> argparse.Namespace:
         )
 
     # Get df of all EPC records with garden size estimates
-    epc_gardens_df = pd.concat(epc_gardens, ignore_index=True)
+    epc_gardens_df = pl.concat(epc_gardens)
     if not args.save_as:
         args.save_as = f"s3://asf-heat-pump-suitability/outputs/{year}Q{q}/{datetime.today().strftime('%Y%m%d')}_{year}_Q{q}_EPC_garden_size_estimates_{args.nations.upper()}.parquet"
-    epc_gardens_df.to_parquet(args.save_as, engine="pyarrow")
+    save_utils.save_to_s3(epc_gardens_df, args.save_as)
+
+    logging.info("Deduplicating UPRNs that were matched to multiple gardens")
```

Comment:
this deduplication was applied in a separate script previously (`run_process_garden_size.py`) but I wanted to put it together